test/protocol/test/textDocument/semanticTokens/full.test.ts

```diff
@@ -29,8 +29,8 @@ describe('[hardhat] semanticTokens/full', () => {
       data: [
         4, 5, 10, 2, 0, 2, 10, 13, 2, 0, 2, 7, 10, 2, 0, 6, 5, 8, 2, 0, 6, 6, 11, 2, 0, 2, 8, 9, 2, 0, 2, 9, 12, 2, 0,
         1, 10, 9, 2, 0, 1, 8, 12, 2, 0, 0, 35, 13, 2, 0, 0, 37, 10, 2, 0, 3, 4, 12, 2, 0, 2, 4, 13, 2, 0, 2, 4, 10, 2,
-        0, 6, 13, 12, 4, 0, 1, 8, 12, 2, 0, 1, 8, 13, 2, 0, 1, 8, 10, 2, 0, 2, 8, 12, 2, 0, 2, 8, 13, 2, 0, 1, 8, 10, 2,
-        0, 10, 13, 9, 2, 0, 4, 8, 10, 2, 0, 2, 8, 15, 4, 0, 3, 13, 15, 4, 0,
+        0, 6, 12, 13, 4, 0, 1, 8, 12, 2, 0, 1, 8, 13, 2, 0, 1, 8, 10, 2, 0, 2, 8, 12, 2, 0, 2, 8, 13, 2, 0, 1, 8, 10, 2,
```

Comment:
Similar to the two tokens changed here.
Can you please tell me how how to interpret these changes/numbers?
Or maybe we can add a comment to this test to document the process..

Reply:
Great observation. It was because I was matching using the `name` label from the `FunctionDefinition` rule. So it matched against `FunctionName` rule, instead of the `Identifier` token inside it, which is the one we were using before. So I just changed it to match the identifier and reverted the test expectation to the previous one

Reply:
Thanks for explaining.
In that case, I think matching against `FunctionName` is more correct, because then the new code will ignore/miss both `FallbackKeyword` and `ReceiveKeyword`?

Reply:
Additionally, I'm not sure I understand how did you debug this to poinpoint `FunctionDefinition`, and this knowledge would be very helpful in the future if anyone needed to update the queries/grammar.

I suggest adding a comment to this test to document your process here.